guides/source/configuring.md

```diff
@@ -946,6 +946,53 @@ Specifies if an error should be raised if the order of a query is ignored during
 
 Controls whether migrations are numbered with serial integers or with timestamps. The default is `true`, to use timestamps, which are preferred if there are multiple developers working on the same application.
 
+#### `config.active_record.db_warnings_action`
+
+Controls the action to be taken when a SQL query produces a warning. The following options are available:
+
+  * `:ignore` - Database warnings wil be ignored. This is the default.
+
+  * `:raise` - Database warnings will be raised as `ActiveRecord::SqlWarning`.
+
+  * `:log` - Database warnings will be logged via `ActiveRecord.logger` at the `:warn` level.
+
+  * Custom proc - A custom proc can be provided. It should accept a `SqlWarning` error object and an
+    `ActiveRecord::DatabaseConfigurations::HashConfig` object.
+
+    For example:
+
+    ```ruby
+    config.active_record.db_warnings_action = ->(warning, db_config) do
+      # Report to custom exception reporting service
+      Bugsnag.notify(warning.message) do |notification|
+        notification.add_metadata(:warning_code, warning.code)
+        notification.add_metadata(:warning_level, warning.level)
+        notification.add_metadata(:connection_name, db_config.name)
```

Comment:
In all other places in Active Record `connection_name` refers to the "class that established the connection". It's the unique key used to find the pool manager. I get that this is just an example but I'm trying to separate the two because it leads to confusion. Can you change this to `db_config_name`? or something like that.

Reply:
Yes! Thank you for clarifying ❤️